packages/react/src/field/FieldLabel.tsx

```diff
@@ -17,14 +18,15 @@ export const FieldLabel = ({
 	required,
 	secondaryLabel: secondaryLabelProp,
 	hideOptionalLabel,
+	className,
 }: FieldLabelProps) => {
 	const secondaryLabel = useMemo(() => {
 		if (hideOptionalLabel) return null;
 		if (secondaryLabelProp) return secondaryLabelProp;
 		if (!required) return `(optional)`;
 	}, [required, secondaryLabelProp, hideOptionalLabel]);
 	return (
-		<Flex as={as} htmlFor={htmlFor} gap={0.25}>
+		<Flex as={as} htmlFor={htmlFor} gap={0.25} className={className}>
```

Comment:
`className` has been added so that we can visually hide the fieldset in the date range picker.